modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -66,37 +66,61 @@ MCO:: Rolls out an update to the systemd configuration necessary for OpenShift S
 
 . Stop all of the machine configuration pools managed by the Machine Config Operator (MCO):
 
-** Stop the master configuration pool:
+** Stop the `master` configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool master --type='merge' --patch \
   '{ "spec": { "paused": true } }'
 ----
 
-** Stop the worker machine configuration pool:
+** Stop the `worker` machine configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused": true } }'
 ----
 
 . To prepare for the migration, set the migration field to `null` by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Check that the migration status is empty for the `Network.config.openshift.io` object by entering the following command. Empty command output indicates that the object is not in a migration operation.
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
+----
+$ oc get Network.config cluster -o jsonpath='{.status.migration}'
+----
+
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
++
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": { "networkType": "OpenShiftSDN" } } }'
+----
++
+[IMPORTANT]
+====
+If you applied the patch to the `Network.config.openshift.io` object before the patch operation finalizes on the `Network.operator.openshift.io` object, the Cluster Network Operator (CNO) enters into a degradation state and this causes a slight delay until the CNO recovers from the degraded state.
+====
+
+. Confirm that the migration status of the network plugin for the `Network.config.openshift.io cluster` object is `OpenShiftSDN` by entering the following command:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get Network.config cluster -o jsonpath='{.status.migration.networkType}'
+----
 
+. Apply the patch to the `Network.config.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
++
+[source,terminal,subs="+quotes"]
+----
 $ oc patch Network.config.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "networkType": "OpenShiftSDN" } }'
 ----
```

Comment:
Completely out of scope and up to you, but the optional step below should have "You can" removed from the beginning of the sentence. If you look at the [ISG examples](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) for optional steps, the "Optional:" does the work of saying "you can", and everything that follows is still written like a normal step in the imperative mood

Reply:
:heavy_check_mark: